### PR TITLE
Fix dynamic partial ellipsoids

### LINKED ISSUE #8271: EllipsoidGraphics.maximumCone cannot be dynamic
在cesium1.62 , EllipsoidGraphics.maximumCone 使用  new Cesium.CallbackProperty不起作用?
例：
var viewer = new Cesium.Viewer('cesiumContainer');
var i = 50;
var entity = viewer.entities.add({
    name : 'Dome with inner radius',
    position: Cesium.Cartesian3.fromDegrees(-114.0, 40.0),
    ellipsoid : {
        radii : new Cesium.Cartesian3(1000, 1000, 1000),
        maximumCone :new Cesium.CallbackProperty(setAngle, false),
        material : Cesium.Color.RED.withAlpha(0.3),
        outline : true
    }
});

function setAngle() {  
    //只走一次
   if(i>0){
     i-=10; 
   }         
   return Cesium.Math.toRadians(i);
}

viewer.zoomTo(viewer.entities);
请问怎么解决这个问题

### PR DESCRIPTION
**Pull request by hpinkos**
Fixes #8271

This was an oversight on my part when I merged the partial ellipsoid PR.  We need to rebuild the ellipsoid geometry when the clock and cone values change for dynamic ellipsoids.

```js
var viewer = new Cesium.Viewer('cesiumContainer', {
    shouldAnimate: true
});

var entity = viewer.entities.add({
    position: Cesium.Cartesian3.fromDegrees(-120.0, 40.0),
    ellipsoid : {
        radii : new Cesium.CallbackProperty(setRadii, false),
        maximumCone :new Cesium.CallbackProperty(setAngle, false),
        material : Cesium.Color.BLUE.withAlpha(0.3),
        outline : true
    }
});

function setRadii(time) {
    var radius = Math.abs(Math.cos(time.secondsOfDay)) * 100000 + 30000.0;
    return new Cesium.Cartesian3(radius, radius, radius);
}

function setAngle(time) {
    var angle = Math.abs(Math.sin(time.secondsOfDay)) * Math.PI;
    return angle;
}

viewer.zoomTo(viewer.entities);
```

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/fix-dynamic-ellipsoid/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/DataSources/EllipsoidGeometryUpdater.js [hunk 1]
```diff
@@ -32,14 +32,15 @@ import Property from './Property.js';
...
+    var innerRadiiScratch = new Cartesian3();
     var scratchColor = new Color();
     var unitSphere = new Cartesian3(1, 1, 1);
 
     function EllipsoidGeometryOptions(entity) {
         this.id = entity;
         this.vertexFormat = undefined;
         this.radii = undefined;
-        this.innerRadii = undefined;
+        this.innerRadii =  undefined;
```
Comment (OmarShehata) [inline Source/DataSources/EllipsoidGeometryUpdater.js]: I think you accidentally added an extra space here?

File: Source/DataSources/EllipsoidGeometryUpdater.js [hunk 2]
```diff
@@ -322,19 +336,21 @@ import Property from './Property.js';
...
+            if (defined(innerRadii)) {
+                if (in3D) {
+                    var mag = Cartesian3.magnitude(radii);
+                    options.innerRadii = Cartesian3.fromElements(innerRadii.x/mag, innerRadii.y/mag, innerRadii.z/mag, options.innerRadii);
+                } else {
+                    options.innerRadii = Cartesian3.clone(innerRadii, options.innerRadii);
+                }
             } else {
-               options.innerRadii = innerRadii;
+                options.innerRadii = undefined;
```
Comment (OmarShehata) [inline Source/DataSources/EllipsoidGeometryUpdater.js]: This also looks like an extraneous space.

Comment (hpinkos) [reply Source/DataSources/EllipsoidGeometryUpdater.js]: This one was actually improperly indented before

**Comment by hpinkos** [comment]
@OmarShehata ready

**Pull request merged**